src/stores/Tweets.ts

```diff
@@ -1,9 +1,59 @@
-import { observable } from 'mobx';
+import { observable, action } from 'mobx';
 import { ITweet } from '../interfaces/tweet';
+import { Fetch } from '../helpers/fetch';
+import { env } from 'src/env/environment';
 
 export class Tweets {
   @observable
   tweets: ITweet[] = [];
+
+  @action
+  fetchTweets() {
+    return Fetch.request(env.securedRoutes + '/posts', { method: 'GET' })
+      .then((response: ITweet[]) => {
+        this.tweets = response;
+        return response;
+      });
+  }
+
+  @action
+  addTweet(authorId: number, content: string) {
+    return Fetch.request(
+      env.securedRoutes + '/add_new_post', 
+      { 
+        method: 'POST',
+        headers: {
+          'Accept': 'application/json',
+          'Content-Type': 'application/json'
+        },
+        body: JSON.stringify({
+          'authorId': authorId,
+          'content': content
+        })
+      }
+    )
+    .then(() => {
+      this.fetchTweets();
+    });
+  }
+
+  @action
+  removeTweet(tweetId: number) {
+    return Fetch.request(
```

Comment:
The same. After deleting you will recive all Tweets from database as response

Reply:
It's not ok to receive the list on the response of request creating a new item, usually in this case response should be with 201 code and/or with an empty body or only with just created item.